docs/datatypes/chain.md

```diff
@@ -1,60 +1,70 @@
 # Chain
 
-`Chain` is a data structure that allows constant time prepending and appending.
-This makes it especially efficient when used as a `Monoid`, e.g. with `Validated` or `Writer`.
-As such it aims to be used where `List` and `Vector` incur a performance penalty.
+@:api(cats.data.Chain) is a data structure that allows constant time prepending and appending.
```

Comment:
This should really be `constant time prepending, appending and concat`. The last bit is important. List and Queue offer constant time prepend and append (respectively) but not concat. This is a key difference here.

Reply:
Perhaps, it would be even better to say something like
> ... constant time concat (as well as append and prepend via concat)

Just to emphasize, that there's no separate append nor prepend under the hood – both are nothing but shortcuts for concat.

Reply:
For the first sentence, I'd rather just state the performance characteristics directly than start addressing implementation details. I've updated to reflect the O(1) concat, and mention that append and prepend are concats in the How it Works section.